layers/+music/tidalcycles/funcs.el

```diff
@@ -0,0 +1,19 @@
+;;; funcs.el --- TidalCycles Layer functions File for Spacemacs
+
+(when (configuration-layer/package-usedp 'tidal)
```

Comment:
This test can be removed, you are loading tidal as part of your package.el so you can expect it to be there.